discovery_engine/lib/src/discovery_engine_base.dart

```diff
@@ -467,6 +470,27 @@ class DiscoveryEngine {
     });
   }
 
+  /// Requests for the currently [TrendingTopic]s.
```

Comment:
```suggestion
  /// Requests for the current [TrendingTopic]s.
```
:art: this is kinda tricky though. i'd choose the adjective as the class name is a noun. if we were to split the class name into a verb and noun, then i'd keep the adverb. not really important after all.